elementary/clients/dbt/dbt_runner.py

```diff
@@ -241,3 +242,28 @@ def ls(self, select: Optional[str] = None) -> list:
 
     def source_freshness(self):
         self._run_command(command_args=["source", "freshness"])
+
+    def get_installed_packages(self):
+        dbt_packages_folder = os.path.join(os.getcwd(), 'dbt_packages')
+        try:
+            folder_names = [name for name in os.listdir(dbt_packages_folder) if os.path.isdir(os.path.join(dbt_packages_folder, name))]
+            return folder_names
+        except FileNotFoundError:
+            logger.info("'dbt_packages' folder not found.")
+            return False
+
+    def _run_deps_if_needed(self):
+        packages_yaml_file = os.path.abspath('packages.yml')
```

Comment:
Same as comment above, join `packages.yml` with the [dbt project's path](https://github.com/elementary-data/elementary/blob/ec17900a8f5712061b32024c600d05c9c9b411fa/elementary/monitor/dbt_project_utils.py#L12).